packaging/ondemand.spec

```diff
@@ -98,11 +98,12 @@ echo "SELinux policy %{selinux_policy_ver}"
 %__cp %{SOURCE3} ./ondemand-selinux.fc
 %__sed -i 's/@VERSION@/%{selinux_module_version}/' ./ondemand-selinux.te
 %__make -f %{_datadir}/selinux/devel/Makefile
+popd
 
 scl enable ondemand - << \EOS
 set -x
 set -e
-rake -mj%{ncpus}
+rake --trace -mj%{ncpus} OBJDIR=$(pwd)/build
```

Comment:
does it matter that `OBJDIR=$(pwd)/build` is after instead of before `rake --trace -mj%{ncpus}`

Reply:
Of course the build completed successfully so I guess not...

Reply:
It sets as environment variable that way, and works.